Updated /slack etiquette section, _slack.scss, and head.html

sendmebiscuits opened the pull request:
Fixes #924 

Updated the content under _What is the Slack Etiquette in this community_ on codebuddies.org/slack to the content from https://slack-files.com/T04AQ6GEY-F8VRAGD0S-ebfb2269d1. Fixed a few typos while I was at it! Updated _slack.scss as well. Also updated the fontawesome CDN in head.html to access relevant emojis. 

If there are any changes you want added, let me know!

lpatmo commented on client/templates/other/slack.html:
```diff
@@ -36,24 +36,93 @@ <h3>Quick Links</h3>
                 <p>We use a bot called plusplus to applaud or encourage each other. For example, if @hermionegranger gives me a helpful answer to a question I asked in a channel, I might type @hermionegranger++.
-                  Think of these points as virtual cookies. :)</p>
+                  Think of these points as virtual cookies. <span style="font-size: 20px"><i class="far fa-smile-wink"></i></span></p>
               <h3>Who can create new channels on Slack?</h3>
```
Sorry I didn't mention it earlier: we can delete these entire question and answer ("Who can create new channels on Slack?") because now anybody can create a public channel again, and the public channel will be posted automatically to #announcements so admins can track it and know if we have a duplicate channel, or something.

lpatmo commented on client/templates/other/slack.html:
```diff
@@ -36,24 +36,93 @@ <h3>Quick Links</h3>
               <h3>How do you show your appreciation to someone on Slack?</h3>
                 <p>We use a bot called plusplus to applaud or encourage each other. For example, if @hermionegranger gives me a helpful answer to a question I asked in a channel, I might type @hermionegranger++.
-                  Think of these points as virtual cookies. :)</p>
+                  Think of these points as virtual cookies. <span style="font-size: 20px"><i class="far fa-smile-wink"></i></span></p>
```
Let's use `fas` or `fab` here, since `far` is technically a PRO logo that we'd need to buy a license for (and PRO licenses aren't recommended for open-sourced proejcts). Let me know if that makes sense. Thanks!

sendmebiscuits replied:
Made perfect sense! 😁

sendmebiscuits commented:
Alright pushed an update. Changed the emoji to fas and deleted the _Who can create a channel section_. Let me know if there's anything else you'd like updated!

lpatmo commented:
@sendmebiscuits Thank you so much!

sendmebiscuits commented:
My pleasure!
